apm-agent-core/src/main/java/co/elastic/apm/agent/metrics/builtin/SystemMetrics.java

```diff
@@ -78,15 +87,21 @@
 
     @Nullable
     private final Method virtualProcessMemory;
+    private final File meminfo;
```

Comment:
```suggestion
    private final File meminfoFile;
```
Just because we have a `memInfo` variable later...